pophory-iOS/Global/Extensions/UIViewController+Extension.swift

```diff
@@ -23,4 +23,14 @@ extension UIViewController {
         self.navigationController?.isNavigationBarHidden = false
     }
     
+    /// 화면밖 터치시 키보드를 내려 주는 메서드
+    func hideKeyboard() {
+        let tap: UITapGestureRecognizer = UITapGestureRecognizer(target: self,
+            action: #selector(UIViewController.dismissKeyboard))
+        view.addGestureRecognizer(tap)
+    }
+    
+    @objc func dismissKeyboard() {
```

Comment:
이건 private으로 선언해주면 좋을 거 같애. 
다른 곳에서 쓰려다가 hideKeyboard() 부르려는 거 잘못해서 dismissKeyboard() 불러버릴 수도 있을 거 같다!
```suggestion
    @objc private func dismissKeyboard() {
```

Reply:
오.. 반영했어!